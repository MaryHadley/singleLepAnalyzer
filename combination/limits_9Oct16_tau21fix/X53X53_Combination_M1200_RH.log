000 (1295364.000.000) 10/09 22:41:04 Job submitted from host: <192.168.63.249:41533>
...
001 (1295364.000.000) 10/09 22:41:26 Job executing on host: <192.168.63.239:46789>
...
006 (1295364.000.000) 10/09 22:41:34 Image size of job updated: 188484
	86  -  MemoryUsage of job (MB)
	87772  -  ResidentSetSize of job (KB)
...
006 (1295364.000.000) 10/09 22:46:34 Image size of job updated: 282008
	161  -  MemoryUsage of job (MB)
	164492  -  ResidentSetSize of job (KB)
...
006 (1295364.000.000) 10/10 17:48:34 Image size of job updated: 297588
	161  -  MemoryUsage of job (MB)
	164492  -  ResidentSetSize of job (KB)
...
005 (1295364.000.000) 10/10 18:01:59 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 19:18:54, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 19:18:54, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1596  -  Run Bytes Sent By Job
	307  -  Run Bytes Received By Job
	1596  -  Total Bytes Sent By Job
	307  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :    31818        1   3824810
	   Memory (MB)          :      161     3072      3072
...
