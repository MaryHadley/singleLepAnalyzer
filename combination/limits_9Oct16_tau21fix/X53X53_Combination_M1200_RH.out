/var/lib/condor/execute/dir_15369
Total number of likelihood evaluations: 1100000101
Total number of likelihood evaluations: 11000101
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1200_RH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1200_RH/quant-toys:0-sig-a83ad13574.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1200_RH/quant-data-sig-5fed5842ae.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1200_RH
