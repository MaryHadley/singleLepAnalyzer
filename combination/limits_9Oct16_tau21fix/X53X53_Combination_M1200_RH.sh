#!/bin/sh 
cd /home/ssagir/CMSSW_7_3_0/src/
source /cvmfs/cms.cern.ch/cmsset_default.sh
cmsenv
cd -
cd limits_9Oct16_tau21fix/
/home/ssagir/CMSSW_7_3_0/src/theta/utils2/theta-auto.py /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1200_RH.py