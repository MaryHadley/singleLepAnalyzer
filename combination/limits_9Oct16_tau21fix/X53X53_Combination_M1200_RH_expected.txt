# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0   0.068931     0.0349    0.14404  0.047718    0.10119
